python/ray/serve/api.py

```diff
@@ -276,14 +284,23 @@ def create_backend(backend_tag: str,
 
     if config is None:
         config = {}
-    if not isinstance(config, dict):
-        raise TypeError("config must be a dictionary.")
+    if not isinstance(config, BackendConfig) and not isinstance(config, dict):
+        raise TypeError("config must be a BackendConfig or a dictionary.")
 
     replica_config = ReplicaConfig(
         func_or_class, *actor_init_args, ray_actor_options=ray_actor_options)
-    backend_config = BackendConfig(config, replica_config.accepts_batches,
-                                   replica_config.is_blocking)
-
+    if isinstance(config, dict):
+        backend_config = BackendConfig(
```

Comment:
let's use `BackendConfig. parse_obj` here so it's more idiomatic https://pydantic-docs.helpmanual.io/usage/models/#helper-functions